tv.kodi.Kodi.json

```diff
@@ -62,6 +62,10 @@
                 "-DLIBDVDNAV_URL=build/download/libdvdnav-6.0.0-Leia-Alpha-3.tar.gz",
                 "-DJava_JAVA_EXECUTABLE=/usr/lib/sdk/openjdk10/bin/java"
             ],
+            "cleanup": [
+                "/include",
```

Comment:
You don't need to repeat cleanup lines in every module. Just put commonly used ones like `/include`, `*.a`, `/lib/pkgconfig` and so on at the root of manifest.

Reply:
that is convenient, but in some cases the /bin/ files need to be kept, and I like knowing what is cleaned from what modules.

maybe later people can reuse these modules as well easier this way

Reply:
I've meant only files which should be cleaned for all modules, e.g. `/include`. Cleanups for `/bin` obviously should be defined per-module.